styleguide/_themes/derek/scss/_modifiers.scss

```diff
@@ -12,3 +14,68 @@
 .bg-white{ background-color: $white;}
 .bg-medium-teal { background: $mediumteal; }
 .bg-crumpled-paper { background-color: $light-gray; background-image: url(https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/elements/bg-crumpled-paper.png); }
+
+//centered-divs
+.center-block{ margin: 0 auto; float:none;}
+
+//equal height columns
+.row-eq-height {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display:         flex;
+}
+
+@media only screen and (max-width : 770px) {
+  //equal height columns remove in small
+  .row-eq-height {
+  display: block;
+  }
+}
+
+//absolutely positioning an element to the bottom of its container
+.bottom{
+  position: absolute;
+  bottom: 0;
+  @extend .full-width;
+  right: 0;
+  height:10%;
+}
+
+.circle{
+  border: solid 2px $darkest-gray;
+  border-radius: 50%;
+  background-color: #fff;
+  padding: 10px;
+  display: inline-block;
+  margin-bottom: 15px;
+}
+.border-white{
+  border:solid 1px $white;
+  padding: 15px;
+}
+
+.clickarea {
+  padding: 5%;
+  background-color:$white;
+  a, a:hover {
+    text-decoration: none;
+    &:hover { text-decoration: none; }
+  }
+  &:hover {
+    background-color: $light-gray;
+    cursor:pointer;
+  }
+}
+
+.river-light {
```

Comment:
@amberRrucker yes this is used to control the inversion of river header text elements:

https://github.com/rackerlabs/zoolander/blob/69dcf65cbf1ad99a4d3cddb81c39c85604c8909d/styleguide/derek/examples/solutions.ejs#L169

Open for refactor. Thanks for asking.